src/withEditorProps/index.js

```diff
@@ -131,6 +131,9 @@ export const importSequenceFromFile = props => (file, opts = {}) => {
         updateSequenceData(result[0].parsedSequence);
         if (!failed) {
           window.toastr.success("Sequence Imported");
+          if (onImport){ 
+            onImport();
```

Comment:
@ericsturman Do you not want to pass the imported sequence data here to this handler? Maybe it wouldn't be useful for your use case right now but it seems like it might be for future users of this function? 

Also it might make sense to have this handler come before the updateSequenceData() call in order to allow the user to block or change import if they wanted in an async manner. I'm thinking something like: 

```js
if (failed) { return window.toastr.error("Error importing sequence")}
const seqDataToImport = await onImport ? onImport(result[0].parsedSequence) : result[0].parsedSequence
if (!seqDataToImport) {
    updateSequenceData(seqDataToImport);
    window.toastr.success("Sequence Imported");
}
```

I just wrote that inline so there might be errors but that is kind of what I'm thinking? Thoughts?